packages/@lightningjs/ui-components/src/components/InlineContent/InlineContent.js

```diff
@@ -360,6 +359,26 @@ export default class InlineContent extends Base {
         text: text.text || text
       }
     };
+    if (textOverrideStyles?.textDecoration === 'line-through') {
+      const textWidth = measureTextWidth({
+        ...this.style.textStyle,
+        ...textOverrideStyles,
+        text: text.text || text
+      });
+      const strikethroughLine = {
+        rect: true,
+        w: textWidth - 2,
+        h: 2,
+        color: textOverrideStyles?.textColor || 0xffffffff,
+        y: textComponent.h / 2
+      };
+      return {
+        type: lng.Component,
+        w: textWidth + 5,
+        children: [{ ...textComponent }, { ...strikethroughLine }]
+      };
```

Comment:
I think we're going to have to workshop this. It seems to be causing the text after it to overlap when `contentWrap` is true.

<img width="68" alt="Screenshot 2024-09-12 at 4 02 49 PM" src="https://github.com/user-attachments/assets/f1adda9a-978e-4e64-bc1c-5d2a2807597a">